velox/common/memory/MemoryPool.cpp

```diff
@@ -116,6 +116,10 @@ void capExceedingMessageVisitor(
     MemoryUsageHeap& topLeafMemUsages,
     std::stringstream& out) {
   const MemoryPool::Stats stats = pool->stats();
+  // Avoid logging empty pools.
+  if (stats.currentBytes == 0 and stats.peakBytes == 0) {
```

Comment:
Nice change!
nit: add a method say MemoryPool::Stats::empty()? Thanks!

Reply:
Yes, we can add this.